plugins/generator-1.21.1/datapack-1.21.1/jsontriggers/item_start_using.json.ftl

```diff
@@ -0,0 +1,8 @@
+"${registryname}_${cbi}": {
+  "trigger": "minecraft:using_item",
+  "conditions": {
+    "item": {
+    	${input$item}

```

Comment:
You have tab here, please, go through changed templates again and check if any more tabs are present like here